deps/glad/CMakeLists.txt

```diff
@@ -30,14 +30,12 @@ if(OS_WINDOWS)
 
 elseif(OS_POSIX AND NOT OS_MACOS)
   find_package(OpenGL REQUIRED)
-  find_package(X11 REQUIRED)
-
-  target_link_libraries(glad PRIVATE X11::X11)
```

Comment:
Is this dep removal possible because of this commit; or is it a separate cleanup? Could you please mention this on the commit message?

Reply:
Oh thanks, this was actually just me testing something out, I meant to revert this.